scripts/idl/matter_idl_types.py

```diff
@@ -21,9 +21,14 @@ def __init__(self, meta: Meta = None, line: int = None, column: int = None):
             self.column = column
 
 
+class StructAttribute(enum.Enum):
+    FABRIC_SCOPED = enum.auto()
```

Comment:
Perhaps our linter should now checks that all Structs that are fabric-scoped have a FabricIndex global field 254

Reply:
Agree, but maybe separate PR?

Was also unsure longer-term if we should have FabricIndex or just make it implied if the struct is marked fabric scoped.